content/tutorials/xrp-ledger-businesses/list-your-exchange-on-xrp-charts.md

```diff
@@ -0,0 +1,197 @@
+
+2. [Provide a Current Order Book RESTful API endpoint](#provide-a-current-order-book-endpoint).
+
+3. [Send an exchange listing request to XRP Charts](#send-an-exchange-listing-request-to-xrp-charts).
+
+Your exchange may have existing RESTful API endpoints that can deliver data in the formats described in this document. If so, there may be little to no development effort required on your part to complete tasks 1 and 2.
+
+If you have any questions about endpoint specifications, contact xxx@xxxxx.com. ***TODO: do we want to provide a way for them to contact us for dev support?***
+
+
+## Provide a Recently Executed Trades Endpoint
+
+Provide a RESTful API endpoint that returns the most recent 500-1,000 individual trades executed in a particular XRP market.
+
+To ensure that it doesn't miss a trade, XRP Charts queries the endpoint frequently, between every 5 and 30 seconds, aiming to get responses that have overlapping trade data. Ensure that any rate limit enforced by your endpoint can accommodate this query frequency. XRP Charts records unique trade data only, even if it gets overlapping trades.
+
+If XRP Charts needs to query your endpoint at a frequency that exceeds your rate limit, XRP Charts may request that you adjust the rate limit or provide the `last_tid` parameter. ***TODO: okay to state this? accurate?***
+
+
+### Request Format
+
+Provide a request format like the following:
+
+```
+GET {api_base_url}/v1/trades
+```
+
+#### Authentication
+
+We prefer to work with a publicly accessible endpoint.
+
+#### Parameters
+
+Provide parameters that help XRP Charts filter trades returned in the response. The parameter field names are examples. You can use other names. Your endpoint doesn't have to provide the optional parameters, but they are useful.
+
+| Field      | Type    | Description                                           |
+|:-----------|:--------|:------------------------------------------------------|
+| `market`   | String  | Returns trades executed in a particular market in which XRP is either the base or counter currency. Use a suggested format of `<basecurrency>_<countercurrency>` to provide valid values like `xrp_btc`, `btc_xrp`, `xrp_usd`, or `usd_xrp`, for example. |
+| `last_tid` | Integer | _(Optional)_ Returns trades executed after a specific trade ID. For example, XRP Charts can use this parameter to get all trades after the last set of trades recorded in site data, ensuring that it has recorded all trades. |
+| `limit`    | Integer | _(Optional)_ Returns no more than a specific number of trades in a response. |
+
+
+### Response Format
+
+A successful response must be a JSON array of objects, one for each trade. The parameter field names are examples. You can use other names. Your endpoint doesn't have to provide the optional parameters, but they are useful.
+
+***TODO: `tid` should be required, is that right? Or is there some other way to determine uniqueness?***
```

Comment:
I don't think a trade has to have a unique ID. It might be that price/amount/timestamp is enough to determine uniqueness or something.

I'm sure having an explicit ID is useful though.